azurerm/data_source_availability_set.go

```diff
@@ -0,0 +1,82 @@
+package azurerm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmAvailabilitySet() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmAvailabilitySetRead,
+		Schema: map[string]*schema.Schema{
+			"resource_group_name": resourceGroupNameForDataSourceSchema(),
+
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.NoEmptyStrings,
+			},
+
+			"location": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"platform_update_domain_count": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"platform_fault_domain_count": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"managed": {
+				Type:     schema.TypeString,
```

Comment:
actually running the test suite for this identified this should be a Bool not a string - I'll push a commit to update that too, I hope you don't mind!

Reply:
Thanks for catching that.  I ran the test suite locally but didn't see it.... so maybe I did it wrong or missed it because I didn't know what I was looking for.